tools/ci/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM oracle/graalvm-ce:20.2.0-java11
+
+USER root
+
+RUN groupadd -g 1001 enso
+RUN useradd --system --create-home --uid 1001 --gid 1001 enso
+
+ADD runner.jar /opt/enso/runner.jar
+ADD runtime.jar /opt/enso/runtime.jar
+
+RUN chown -hR enso:enso /opt/enso
+RUN chmod -R u=rX,g=rX /opt/enso
+
+RUN mkdir -p /home/enso/workspace
+RUN chown -hR enso:enso /home/enso/workspace
+RUN chmod -R u=rwX,g=rwX /home/enso/workspace
+
+USER enso:enso
+
+WORKDIR /opt/enso
+
+ENTRYPOINT ["java", "-jar",  "-Dtruffle.class.path.append=runtime.jar", "runner.jar" ]
```

Comment:
These set of options may change between versions, maybe it should be documented somewhere that it should be kept in sync with `distribution/manifest.template.yaml`?

Currently `-Dpolyglot.engine.IterativePartialEscape=true` is also being added, but that may change.

Reply:
done